tests/integration/pilot/common/routing.go

```diff
@@ -1160,6 +1162,111 @@ spec:
 	return cases
 }
 
+// consistentHashCases tests destination rule's consistent hashing mechanism
+func consistentHashCases(apps *EchoDeployments) []TrafficTestCase {
+	cases := []TrafficTestCase{}
+	for _, c := range apps.PodA {
```

Comment:
nti: can we use `echotest` here and filter out some clients? I don't have a super strong preference, it's just less work later to make sure this works with different clients (e.g. proxyless gRPC)

Reply:
on second thought, I don't even know if consistentHash works for gRPC? this is fine.